pillar.example

```diff
@@ -19,3 +19,38 @@ docker:
     all: true # true (Default): Remove all unused images, translates to `docker system prune --force --all`
               # false: (Remove only dangling images, translates to `docker system prune --force`
     volumes: false # Remove unused volumes as well. Default: false
+
+
+# Registry credentials for the docker.login state (if used)
+#
+# Note that multiple dictionaries, and multiple registries per dictionary, can be specified:
+#
+
+docker-registries:
+  'https://docker-registry.work.com':
+    username: jrabbit
+    password: s3cr3t
+
+my-private-docker-registries:
+  'https://docker-A.jack-rabbit.com':
+    username: jack
+    password: c4rr0t
+  'https://docker-B.jack-rabbit.com':
+    username: jack
+    password: c4rr0t
+
+
+# Configuration for the docker.machine substate
+#
+# See README on how to obtain the contents of the key and certificate files
+#
+docker-machine:
+  version: v0.16.2-gitlab.12
+  ca-key: |
+    <PGP encrypted contents of /root/.docker/machine/certs/ca-key.pem private RSA key>
```

Comment:
The decision to use GPG to encrypt credentials should be left to the user (even though it's definitly advisable in many cases).

I ususally just insert a dummy cert file that looks a bit like the originals, [like this](https://github.com/chr4/salt-postgresql/blob/master/pillar.example#L48-L66). I think this is a bit easier to understand.

Reply:
adjusted the examples accordingly

Reply:
I think you can leave out the `PGP` hints alltogether, as this seems like we're doing something with the content that's GPG related - but we're not, it's just that it's advisable to provide it with the YAML GPG option.

What would be good is a comment that says that `key` and `ca-key` are considered sensitive and it might be advisable to use [Salt's GPG renderer] feature for encryption.

Reply:
1. the PGP hints were already removed in a previous commit
2. Added the requested comment that suggests encryption

Reply:
You're right. Github diff tricket me there. All good!